pkg/gadgets/trace/network/types/types.go

```diff
@@ -18,16 +18,17 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	PktType string `json:"pktType,omitempty"`
-	Proto   string `json:"proto,omitempty"`
+	PktType string `json:"pktType,omitempty" column:"type,maxWidth:9"`
+	Proto   string `json:"proto,omitempty" column:"proto,maxWidth:6"`
 	IP      string `json:"ip,omitempty"`
```

Comment:
Please also define the column for the IP address but with `hide`. BTW, We are using IP to describe if it is IPv4 or IPv6 in tcp/tcpconnect gadget. Instead, we use saddr or daddr or addr for the IP address (e.g. in bind, tcp, tcpconnect). I think this gadget should align with them: Use `Addr` instead of `IP` for the IP address.

BTW, we already define a sort of template for IP addresses:

```golang
// Min: XXX.XXX.XXX.XXX (IPv4) = 15
// Max: 0000:0000:0000:0000:0000:ffff:XXX.XXX.XXX.XXX (IPv4-mapped IPv6 address) = 45
Addr     string `json:"addr,omitempty" column:"addr,minWidth:15,maxWidth:45"`
```

Reply:
I aligned naming in I did it in https://github.com/inspektor-gadget/inspektor-gadget/pull/1050.

Please use `template:ipaddr` now that it has been [merged](https://github.com/inspektor-gadget/inspektor-gadget/pull/1048).